src/Handlebars.Net.Helpers/HandlebarsHelpers.cs

```diff
@@ -13,7 +13,7 @@
 using HandlebarsDotNet.Helpers.Utils;
 using Stef.Validation;
 using HandlebarsDotNet.Helpers.Models;
-#if NETSTANDARD1_3_OR_GREATER || NET46_OR_GREATER
+#if NETSTANDARD1_3_OR_GREATER || NET46_OR_GREATER || NET6_0_OR_GREATER

```

Comment:
Can this be simplified to be #if !NETFRAMEWORK

Reply:
I don't think so because NET46 is defined as NETFRAMEWORK.